aip/general/0009.md

```diff
@@ -120,12 +120,16 @@ A Network API exposed by a Google service. Most of these are hosted on the
 `googleapis.com` domain. It does not include other types of APIs, such as client
 libraries and SDKs.
 
-### IaC
+### State-driven Clients
```

Comment:
see comment on old PR saying that this should align with an industry standard term. If IaC isn't a good description, how about declarative?

Also note when this is renamed, all references to that term must also be renamed. Please do that in this PR as well.

Reply:
Renamed all to declarative clients